tests/frontend/pages/wallets/wallets.py

```diff
@@ -52,3 +107,18 @@ def delete(self, name):
         buttons = self.driver.find_elements_by_class_name("v-btn")
         submit_button = [button for button in buttons if button.text == "SUBMIT"][0]
         submit_button.click()
+
+    def get_balance(self, name):
+        wallets = self.list()
+        if name in wallets.keys():
+            wallet_card = wallets[name]
+            wallet_card.click()
+            wallet_details = self.driver.find_element_by_class_name("v-data-table")
+            # get spams that contain balances and check that there are spams elements
+            balances_row = wallet_details.find_elements_by_class_name("v-chip__content")
+            if len(balances_row) >= 2:
+                balances = [balances_row[0].text, balances_row[1].text]
+                buttons = self.driver.find_elements_by_class_name("v-btn")
+            close_button = [button for button in buttons if button.text == "CLOSE"][0]
+            close_button.click()
+            return balances
```

Comment:
In case there is no balance, `balances` will not be defined.

Reply:
what should return when no balance ?

Reply:
Empty list I think?